frontend/src/app/app-routing.module.ts

```diff
@@ -1,7 +1,14 @@
 import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router';
+import {CreditsComponent} from './game/credits/credits.component';
+import {FunctionalityInfoComponent} from './game/functionality-info/functionality-info.component';
```

Comment:
It is kind of a good practice to move "page components" to different folder than other components.